package.json

```diff
@@ -19,8 +19,8 @@
     "lint": "eslint --ignore-pattern src/foundationdb/ --ignore-pattern tests/foundationdb/ .",
     "lint-fdb": "eslint .",
     "setversion": "npm run --workspaces setversion && grunt setversion",
-    "test": "npm run build && npm run --workspaces test && npx prisma generate --schema tests/prisma/schema.prisma && jest --coverage --collectCoverageFrom='src/**/*' --collectCoverageFrom='!src/foundationdb/**/*' --detectOpenHandles --testPathIgnorePatterns=examples/* --testPathIgnorePatterns=packages/* --testPathIgnorePatterns=tests/foundationdb/*",
-    "test-fdb": "npm run build-fdb && npm run --workspaces test && npx prisma generate --schema tests/prisma/schema.prisma && jest --runInBand --coverage"
+    "test": "npm run build && rm -rf packages/create/templates/hello-prisma/node_modules/ && npm run --workspaces test && npx prisma generate --schema tests/prisma/schema.prisma && jest --coverage --collectCoverageFrom='src/**/*' --collectCoverageFrom='!src/foundationdb/**/*' --detectOpenHandles --testPathIgnorePatterns=examples/* --testPathIgnorePatterns=packages/* --testPathIgnorePatterns=tests/foundationdb/*",
```

Comment:
What's going on here?

Reply:
Fixing the issue where the Prisma test sometimes picks up the Prisma template--Prisma isn't meant to be used with multiple clients.